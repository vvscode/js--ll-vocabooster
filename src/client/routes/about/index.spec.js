import { equal } from 'assert';
import index from './index';

describe('About/index', () => {
  it('contains action', () => equal(typeof index, 'function'));
  it('action returns object', () => {
    const result = index();
    expect(result.title).toBeDefined();
    equal(typeof result.component, 'function');
  });
  it('action().componnet returns React.Component', () => {
    expect(index().component()).toMatchSnapshot();
  });
});
