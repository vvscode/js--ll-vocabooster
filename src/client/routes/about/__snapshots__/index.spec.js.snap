// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About/index action().componnet returns React.Component 1`] = `
<AboutComponent
  title="About"
/>
`;
